packages/cli/test/models/compiler/solidity/SolidityContractsCompiler.test.js

```diff
@@ -0,0 +1,344 @@
+import CaptureLogs from "../../../helpers/captureLogs";
+
+require('../../../setup')
+
+import { FileSystem } from 'zos-lib'
+import SolidityContractsCompiler from '../../../../src/models/compiler/solidity/SolidityContractsCompiler'
+
+describe('SolidityContractsCompiler', function () {
+
+  const contract_Solc04 = {
```

Comment:
I don't feel comfortable with having so many constant objects declared here. We should move them to another file, or at least move them outside the `describe` scope if we are not saving them in `this`.